app/views/components/_heading.html.erb

```diff
@@ -0,0 +1,11 @@
+<%
+  id = "id=#{id}" if id
+  heading_level ||= 1
```

Comment:
Should we default to 2 since we can make the assumption that we'll only want one H1 and the title component will do that most of the time?